pkg/cidata/cidata.TEMPLATE.d/lima.env

```diff
@@ -1,3 +1,4 @@
+LIMA_CIDATA_NAME={{ .Name }}
```

Comment:
Please add this to the ""Environment variables" section in `docs/internal.md`

Reply:
Added